README.md

```
@@ -1,364 +1,63 @@
+| Platform | Support |
+|----------|---------|
+| Python | Python 3.7+ through 3.13 |
 
-```
-git clone --recurse-submodules https://github.com/ably/ably-python.git
-cd ably-python
-python setup.py install
-```
+> [!NOTE]
+> This SDK works across all major operating platforms (Linux, macOS, Windows) as long as Python 3.7+ is available.
 
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote
null

(MD028, no-blanks-blockquote)